src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -0,0 +1,160 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.Objects;
+
+public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> first;
+    private Node<T> last;
+
+    private int countOfNodes = 0;
+
+    private class Node<N> {
+        private N item;
```

Comment:
@DemPaul we are not pushing the solution! Please remove it. The solution should be provided in the separate file and shared with @Sofasmile via slack